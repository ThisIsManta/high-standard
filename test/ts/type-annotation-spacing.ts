const v1 = 1 as any

const v2:string = 'bar'

function f1():void { }

const f2 = ():string => 'bar'

class MyClass {
  name: string
}

type T = ()=>{}
