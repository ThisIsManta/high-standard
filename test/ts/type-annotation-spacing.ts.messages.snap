error 3:9 [@stylistic/type-annotation-spacing] Expected a space after the ':'.
error 5:14 [@stylistic/type-annotation-spacing] Expected a space after the ':'.
error 7:14 [@stylistic/type-annotation-spacing] Expected a space after the ':'.
error 13:12 [@stylistic/type-annotation-spacing] Expected a space after the '=>'.
error 13:12 [@stylistic/type-annotation-spacing] Expected a space before the '=>'.